resources/gen-ai/aws-rag-appsync-stepfn-opensearch/schema.graphql

```diff
@@ -1,11 +1,20 @@
 type FileStatus @aws_iam @aws_cognito_user_pools {
 	name: String
 	status: String
+	imageurl: String
+}
+
+input ModelConfiguration {
+	provider: String
+	modelId: String
+	streaming: Boolean
```

Comment:
does it make sense to have this option here ? streaming is likely for end user response generation

Reply:
This was an existing attribute from previous schema.